platform/engineering/scheduled-downtime-process.md

```diff
@@ -0,0 +1,10 @@
+# Scheduled Downtime Process for VA.gov
+
+## Overview
+Any site upgrades, migration, or changes that affect will resolute in the public site to experience downtime, should follow the process below in order to allow the right parties to be aware, give enough time for other teams to make any preparations, and to minimize any effect to the Veterans. 
+
+## Steps Needed to be Taken to Schedule Downtime On VA.gov
+
+- Any changes that will result in downtime for the public website need to be communicated with your DEPO lead 2 Weeks before any planned downtime.
```

Comment:
Add first bullet:
- Planned downtime should be added to the downtime schedule (link), and the head of platform should review your addition to the downtime schedule